codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/awssdk/extensions/DafnyPythonAwsSdkSymbolVisitor.java

```diff
@@ -0,0 +1,50 @@
+package software.amazon.polymorph.smithypython.awssdk.extensions;
+
+import static java.lang.String.format;
+
+import software.amazon.polymorph.smithypython.awssdk.AwsSdkCodegenConstants;
+import software.amazon.polymorph.smithypython.common.nameresolver.SmithyNameResolver;
+import software.amazon.polymorph.smithypython.localservice.extensions.DafnyPythonLocalServiceSymbolVisitor;
+import software.amazon.smithy.model.Model;
+import software.amazon.smithy.python.codegen.PythonSettings;
+
+/**
+ * SymbolVisitor for wrapped localService codegen. Overrides the generated file for codegen to
+ * something that is immediately deleted. Smithy ALWAYS writes visited symbols to a file. For AWS
+ * SDK codegen, we do NOT want to write visited symbols to a file, since boto3 does not use these
+ * visited symbols. It is very, very difficult to change this writing behavior without rewriting
+ * Smithy logic in addition to Smithy-Python specific logic. I have tried some workarounds like
+ * deleting writers or writing to /dev/null but these were not fruitful. This workaround dumps any
```

Comment:
commentary - this is annoying, if the API asked for a `File` instead of a filename, we could use something like [`createTempFile`](https://docs.oracle.com/javase%2F8%2Fdocs%2Fapi%2F%2F/java/io/File.html#createTempFile-java.lang.String-java.lang.String-) but alas.